src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/AnnotatedVariantProducer.java

```diff
@@ -108,9 +113,31 @@ static VariantContext produceAnnotatedVcFromInferredTypeAndRefLocations(final Si
         if (altHaplotypeSeq!=null)
             vcBuilder.attribute(GATKSVVCFConstants.SEQ_ALT_HAPLOTYPE, new String(altHaplotypeSeq));
 
+        if (broadcastCNVCalls != null && end > 0) {
+            annotateWithCNVCalls(refLoc, end, vcBuilder, broadcastSequenceDictionary, broadcastCNVCalls);
+        }
+
         return vcBuilder.make();
     }
 
+    private static void annotateWithCNVCalls(final SimpleInterval refLoc,
```

Comment:
a refactoring suggestion: can you make it return a `String` attribute and not taking (hence modifying) a VCBuilder?

Reply:
done